app/components/cookie_banners_component.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= govuk_cookie_banner(
+      title: 'Cookies on Apply for teacher training',
```

Comment:
Should this use the `service_name` helper?

```suggestion
      title: "Cookies on #{service_name}",
```

Reply:
👍